src/conn/opts.rs

```diff
@@ -424,19 +424,19 @@ impl Opts {
 /// Provides a way to build [`Opts`](struct.Opts.html).
 ///
 /// ```ignore
-/// let mut ssl_opts = SslOpts::new();
-/// ssl_opts.set_pkcs12_path("/foo/cert.p12")
-///         .set_root_ca_path("/foo/root_ca.der");
```

Comment:
Same here. `.set_root_ca_path()` doesn't exist, so we need to use `.with_root_ca_path()`.